Tests/scripts/collect_tests/constants.py

```diff
@@ -122,3 +121,13 @@
     FileType.MODELING_RULE_SCHEMA,
     FileType.MODELING_RULE_TEST_DATA,
 }
+
+XSIAM_COMPONENT_FILES: set[FileType] = {
+    FileType.PARSING_RULE,
+    FileType.CORRELATION_RULE,
+    FileType.XSIAM_DASHBOARD,
+    FileType.XSIAM_REPORT,
+    FileType.TRIGGER,
+    FileType.MODELING_RULE_SCHEMA,
+    FileType.XDRC_TEMPLATE
```

Comment:
Is there already `FileType.LayoutRule`? If yes please add it here, if not, @eyalpalo when it will be merged?

Reply:
seems like he didn't merge yet
https://github.com/demisto/demisto-sdk/pull/2640/files#diff-555c51fe59132215b77801a3bb499d421ebd91468c9d7806cedced34215a3048

Reply:
added it :)